seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/TestContainer.java

```diff
@@ -55,4 +55,6 @@ default Container.ExecResult restoreJob(String confFile, String jobId)
     }
 
     String getServerLogs();
+
+    void copyFileMyselfContainers(String path, String targetPath);
```

Comment:
There is no method to copy the file to the seatunnel itself, so an additional one is added

Reply:
```suggestion
    void copyFileToContainer(String path, String targetPath);
```